ci/provisioning/post_provision_config_nodes_UBUNTU_20_04.sh

```diff
@@ -37,12 +39,11 @@ post_provision_config_nodes() {
         done
     fi
     apt-get update
-    if [ -n "$INST_RPMS" ]; then
-        # shellcheck disable=SC2086
-        if ! apt-get -y remove $INST_RPMS; then
+    if [ ${#inst_rpms[@]} -gt 0 ]; then
+        if ! apt-get -y remove "${inst_rpms[@]}"; then
             rc=${PIPESTATUS[0]}
             if [ $rc -ne 100 ]; then
-                echo "Error $rc removing $INST_RPMS"
+                echo "Error $rc removing $INST_RPMS
```

Comment:
(lint) The mentioned syntax error was in this simple command. [SC1009]
(lint) Did you forget to close this double quoted string? [SC1078]